harmony/harmony.py

```diff
@@ -620,6 +617,38 @@ def error_messages(self) -> List[str]:
         return []
 
 
+class JobsRequest(BaseRequest):
+    """A Harmony request to list or search for jobs.
+
+    Args:
+        page (int): The current page number.
+        limit (int): The number of jobs in each page.
+        labels (List[str]): A list of labels to search jobs.
```

Comment:
this is not a parameter (yet)

Reply:
yes, probably should have not added it. It will be used in the next PR when we add search.